Dockerfile_mbtci_template

```diff
@@ -188,15 +235,14 @@ RUN set -ex \
   && tar -xzf apache-maven-${MAVEN_VERSION}-bin.tar.gz -C ${MAVEN_HOME} --strip-components=1 \
   && rm -rf "$GNUPGHOME" apache-maven-${MAVEN_VERSION}-bin.tar.gz.asc apache-maven-${MAVEN_VERSION}-bin.tar.gz \
   && chmod -R a+w ${MAVEN_HOME}/conf/* \
-  && ln -s ${MAVEN_HOME}/bin/mvn /usr/bin/mvn \
```

Comment:
cf. previous comment.

Reply:
OK, delete $MAVEN_HOME from path and revert "ln -s ${MAVEN_HOME}/bin/mvn /usr/bin/mvn"

https://github.com/SAP/cloud-mta-build-tool/pull/1023/commits/b3ea0b94387d71af5ac4e4952a133f0a78328619